### DISCUSSION TITLE
Adding users to bunkered / sandboxed Vaultwarden - no Internet or outgoing SMTP email server

### DISCUSSION THREAD

**Discussion by tfboy** [dani-garcia/vaultwarden#4724] [Q&A] [answered]
TLDR: I have an "off-grid" instance I need to be able to create additional users and then have them added to an organisation. Is this possible?

I have a bunkered installation which is working great for my single admin user created at startup. I now want to add more users.
With the official bitwarden instance, you can work around the email confirmation step by modifying the database to "confirm" email addresses without users actually having received an email and clicked on the link.
But BW uses MSSQL and with VW, it's the far simpler single-container approach.
I've done a quick search but haven't found anything.
Thanks!

**Comment by BlackDex** [comment, marked as answer]
Just do not configure an SMTP server, and  your done :).
You can still limit invites to either a specific user or specific domains.

https://github.com/dani-garcia/vaultwarden/blob/d04b94b77d95a9968f61f71969f65aaaa8a496d6/.env.template#L250-L251

https://github.com/dani-garcia/vaultwarden/blob/d04b94b77d95a9968f61f71969f65aaaa8a496d6/.env.template#L236-L238

Without SMTP enabled, users are set to `accepted` by default, you still need to confirm the users after they have created an account though.
